<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="147762" started-at="2022-02-13T12:40:11 IST" finished-at="2022-02-13T12:42:39 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="147762" started-at="2022-02-13T12:40:11 IST" finished-at="2022-02-13T12:42:39 IST">
      <class name="com.tests.XceliteFeatureTest">
        <test-method status="FAIL" signature="runXceliteDataMappingintoPOJOTest()[pri:0, instance:com.tests.XceliteFeatureTest@71c3b41]" name="runXceliteDataMappingintoPOJOTest" duration-ms="146082" started-at="2022-02-13T12:40:12 IST" finished-at="2022-02-13T12:42:38 IST">
          <exception class="java.util.NoSuchElementException">
            <full-stacktrace>
              <![CDATA[java.util.NoSuchElementException
at java.util.TreeMap$PrivateEntryIterator.nextEntry(Unknown Source)
at java.util.TreeMap$ValueIterator.next(Unknown Source)
at com.ebay.xcelite.reader.BeanSheetReader.read(BeanSheetReader.java:78)
at com.utils.XceliteUtils.readSheet(XceliteUtils.java:32)
at com.utils.XceliteUtils.readSheet(XceliteUtils.java:36)
at com.beans.excel.ExcelToBeanMapper.mapData(ExcelToBeanMapper.java:17)
at com.beans.excel.ExcelToBeanMapper.printPOJOValuesForExcel(ExcelToBeanMapper.java:23)
at com.tests.XceliteFeatureTest.runXceliteDataMappingintoPOJOTest(XceliteFeatureTest.java:13)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.util.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runXceliteDataMappingintoPOJOTest -->
      </class> <!-- com.tests.XceliteFeatureTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
